Workbooks/Storage/Overview/Overview.workbook

```diff
@@ -451,10 +454,10 @@
       "conditionalVisibility": null,
       "name": "text - 4"
     },
-    {
+       {
       "type": 1,
       "content": {
-        "json": "## Information about storage account _{ResourceId:name}_"
+        "json": "### Storage account details: <span style=\"color:blue\"> _{ResourceId:name}_ </span> "
```

Comment:
why is it blue?  on dark theme it will probably be mostly invisible, blue on black background?